app/views/landing/_header.html.erb

```diff
@@ -0,0 +1,54 @@
+<div class="top-contain">
+	<div id="top-fixed">
+		<div class="top-bar valign-wrapper">
+			<div class="container-trim">
+				<div class="left">
+					<span class="white-text small-text">FAST & FREE SHIPPING ON ALL ORDERS - CALL US : 08020000000</span>
+				</div>
+				<div class="right">
+					<%= link_to "About us", "#", class: 'white-text small-text' %>
+					<span class="white-text small-text">&nbsp;|&nbsp;</span>
+					<%= link_to "Blog", "#", class: 'white-text small-text' %>
+					<span class="white-text small-text">&nbsp;|&nbsp;</span>
+					<%= link_to "Contact Us", "#", class: 'white-text small-text' %>
+					<span class="white-text small-text">&nbsp;|&nbsp;</span>
+					<%= link_to "FAQ", "#", class: 'white-text small-text' %>
+					<span class="white-text small-text">&nbsp;|&nbsp;</span>
+					<%= link_to "My Wishlist", "#", class: 'white-text small-text' %>
+					<span class="white-text small-text">&nbsp;|&nbsp;</span>
+					<%= link_to "Currency - NGN", "#", class: 'white-text small-text' %>
+					
+				</div>
+		
+			</div>
+		</div>
+		<ul class="dropdown-content" id="cat_dropdown">
+			<li>Fashion</li>
+			<li>Electronics</li>
+			<li>Home and Kitchen</li>
+			<li>Baby, Kids and Toys</li>
+		</ul>
```

Comment:
Not every page will run my landing controller, as that's where the variable is defined.
So that might not work out fine. The only other option I can think of is to include a @categories variable in each method handling page routes.